test/s3-benchmarks/.scripts/README.md

```diff
@@ -0,0 +1,56 @@
+# Benchmark scripts
```

Comment:
Nice, can we move it to the top level README or link to this from there?

Reply:
done